pkg/util/lifted/visitpod.go

```diff
@@ -241,3 +241,10 @@ func visitContainerConfigmapNames(container *corev1.Container, visitor Visitor)
 	}
 	return true
 }
+
+// VisitPodServiceAccountNames invokes the visitor function with the name of ServiceAccount
```

Comment:
Does this method lifted from kubernetes? If so, we need to add the lift info like this:

```
// +lifted-source: https://github.com/kubernetes/kubernetes/blob/release-1.23/pkg/api/v1/pod/util.go#L245-L261
```

And then run `hack/update-lifted.sh` to update the lifted doc.

Reply:
> Does this method lifted from kubernetes?

no,

Reply:
If not copied from Kubernetes, so this method shouldn't be here.